app/akka/agent/Agent.scala

```diff
@@ -37,42 +37,34 @@ object Agent {
 
     def get(): T = ref.single.get
 
-    def send(newValue: T): Unit = withinTransaction(new Runnable { def run = ref.single.update(newValue) })
```

Comment:
I would not run any IDEA intention 'improvements' on this file so it remains easy to see whether this has diverged or not from the original.

Reply:
Perhaps disable all intentions and you might like to link to the source as well.